theme_kiddo/static/src/scss/primary_variables.scss

```diff
@@ -140,13 +140,10 @@ $o-website-values-palettes: (
         'buttons-font': 'Varela Round',
         'link-underline': 'never',
         'header-template': 'default',
-        'footer-template': 'descriptive',
-        'btn-primary-outline': true,
+        'footer-template': 'centered',
     ),
 );
 
-$o-selected-color-palettes-names: append($o-selected-color-palettes-names, 'kiddo-3');
```

Comment:
Is this not necessary anymore?

Reply:
The palette is duplicated when we use this line with a default palette.